ui/apps/platform/src/Containers/NetworkGraph/components/stylesComponentFactory.tsx

```diff
@@ -1,14 +1,14 @@
 /* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
👀  I might investigate what it would take to turn on no **implicit** any as well, which IMO is arguably worse.

I wonder if there is any way to highlight "contaminated" code that uses variables defined as `any` in the IDE so that we can be extra careful in these rare circumstances.

Reply:
Super thought! Can you add to agenda for team to discuss either Friday or Monday?